qiskit_optimization/problems/quadratic_program.py

```diff
@@ -105,6 +107,9 @@ def __init__(self, name: str = "") -> None:
         self._quadratic_constraints = []  # type: List[QuadraticConstraint]
         self._quadratic_constraints_index = {}  # type: Dict[str, int]
 
+        self._indicator_constraints = []  # type: List[IndicatorConstraint]
```

Comment:
Just another note going forwards. Since we dropped Python 3.5 support, where the typehints had to be done as comments, since 3.6 in the minimum now we can do typehints directly on the members so we don't have to do it via comments anymore.